PxWeb/Code/Api2/DataSelection/ISelectionHandler.cs

```diff
@@ -5,7 +5,6 @@ namespace PxWeb.Code.Api2.DataSelection
 {
     public interface ISelectionHandler
     {
-        public bool Verify(PXModel model, VariablesSelection? variablesSelection, out Problem? problem);
-        public Selection[] GetSelection(PXModel model, VariablesSelection? variablesSelection);
+        public Selection[]? GetSelection(IPXModelBuilder builder, PXModel model, VariablesSelection? variablesSelection, out Problem? problem);
```

Comment:
You should probably remove the model from the arguments and use to model on the builder

Reply:
Now uses builder.Model for the PxModel